app/lib/actions/katello/content_view/remove.rb

```diff
@@ -67,7 +67,7 @@ def plan(content_view, options)
 
             if organization_destroy
               content_view.hostgroups.clear
```

Comment:
This should likely be destroy_all as well

Reply:
I'm not against changing it but I think my test illustrates that at least for now w/ the way the associations are set up, we don't need it. That said, will update!

Reply:
I think we would have run into a problem if something else referenced the hostgroup that would have been 'deleted'.  Such as another hostgroup or another host that hadn't been deleted yet